samples/helloworld.c

```diff
@@ -0,0 +1,8 @@
+// Traditional "Hello World" program
+
+#include <stdio.h>
+
+int main() {
```

Comment:
We don't want "old fashioned" C. Please use `int main(void)`.

Reply:
Very well.  Change made.